docs/source/chatops/aliases.rst

```diff
@@ -103,11 +104,12 @@ Using this example:
 .. code-block:: yaml
 
     formats:
-      - "google {{query=StackStorm}}"
+      - "run {{cmd}} on {{hosts=localhost}}"
 
-In this case the query has a default value assigned which will be used if no value is provided by the user.
-Therefore, a simple ``google`` instead of ``google StackStorm`` would result in assigning the
-default value, in a similar manner to how Action default parameter values are interpreted.
+In this case the query has a default value assigned which will be used
+if no value is provided by the user. Therefore, a simple ``run date`` instead of
+``run date on localhost`` would result in assigning the default value, in a similar
+manner to how Action default parameter values are interpreted.
```

Comment:
> Therefore, a simple `run date` instead of `run date on localhost` would result in assigning the default value

It won't. `run date on` will, because the "on" part is not optional. :) I'll change it to something slightly different.